__tests__/app.test.js

```diff
@@ -31,3 +30,18 @@ describe("/api/topics", () => {
     });
   });
 });
+describe("/api", () => {
+  describe("GET", () => {
+    test("status: 200 responds with an object describing all available endpoints", () => {
+      return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({body}) => {
+        expect(typeof body).toBe("object")
+        expect(Object.keys(body).length).toBeGreaterThan(0)
+        expect(JSON.stringify(body)).toEqual(JSON.stringify(original))
+      });
+    });
+  });
+});
```

Comment:
This test is fine Hannah well done, requiring in the json file and checking the response body looks like the json file is good.